docs/sources/tempo/operations/prometheus_datasource.md

```diff
@@ -0,0 +1,104 @@
+---
+aliases: []
+title: Prometheus compatibility
+menuTitle: Prometheus compatibility
+description: Learn about using tempo as a Prometheus data source.
+weight: 550
```

Comment:
This page weight will put the page at the very bottom of the Manage TOC, underneath Generic forwarding. is that okay?  If so, then we're good to merge. 
![image](https://github.com/grafana/tempo/assets/104772500/52f95c08-ec02-401b-87dd-a51d7470ab9d)